http_stubs/views.py

```diff
@@ -35,14 +36,21 @@ def _httpstub_executor(
     """
     sleep(stub.resp_delay / 1000)
 
+    request_body = _request_body_decode(request)
+
+    tpl_ctx = {'body': request_body}
+
+    for arg_name, arg_val in dict(request.GET).items():
```

Comment:
```suggestion
    for arg_name, arg_val in request.GET.items():
```
The difference is only in handling keys with multiple values, and I don't know which behavior is more correct.